### PR TITLE
Mod 9968 tooltip doesn't open on mouse click

### PR DESCRIPTION
**Pull request by chasls2**
**High level description:**

High level description of the PR.

**Technical details:**

Technical details for the knowledge of other developers.

**JIRA Ticket:**
[DEV-9968](https://federal-spending-transparency.atlassian.net/browse/DEV-9968)

**Mockup:**
https://invis.io/RYA3XN5WP#/273832670_Homepage_2-2_E

The following are ALL required for the PR to be merged:

Author:
- [x] Linked to this PR in JIRA ticket
- [ ] Scheduled demo including Design/Testing/Front-end OR Provided instructions for testing in JIRA and PR `if applicable`
- [x] Verified cross-browser compatibility: Chrome, Safari, Firefox, Edge
- [x] Verified mobile/tablet/desktop/monitor responsiveness
- [x] Verified that this PR does not create any *new* accessibility issues (via Axe Chrome extension)
- [ ] Added Unit Tests for helper functions, reducers, models and Container/Component Interactivity Expectations `if applicable` [React Testing Library](react-testing-library.md)
- [ ] [API contract](https://github.com/fedspendingtransparency/usaspending-api/tree/dev/usaspending_api/api_contracts) updated `if applicable`
- [ ] [Component Library Integration Status Report](https://github.com/fedspendingtransparency/data-act-documentation/blob/data-transparency-ui/frontend_apps/component-library-integration-status.md) updated `if applicable`

Reviewer(s):
- [ ] Design review complete `if applicable`
- [ ] [API #1234](https://github.com/fedspendingtransparency/usaspending-api/pull/1234) merged concurrently `if applicable`
- [ ] Code review complete

### CODE REVIEW

File: src/_scss/pages/explorer/detail/header/header.scss [hunk 1]
```diff
@@ -79,7 +79,7 @@
 
         .detail-header__icon {
           padding: 0 0 0 rem(5);
-          cursor: default;
+          cursor: pointer;
```
Comment (brianpetway) [inline src/_scss/pages/explorer/detail/header/header.scss:82]: this can remain as cursor: default; our other tooltips don't have pointer

Comment (chasls2) [reply src/_scss/pages/explorer/detail/header/header.scss]: Done

**Comment by brianpetway** [comment]
You achieved the main goal of getting the tooltip to show on click/tap at tablet and mobile, but now the way the tooltip closes is different at desktop, and not as good.
I think this requires a bit more time to figure out how to keep the behaviors of both the onMouseLeave and onBlur in RootHeader, while still getting the tooltip to show at tablet and mobile.
It would also be good to adjust the position of the tooltip at tablet and mobile so that it doesn't cover the info icon, even though that wasn't in the ticket specifically.
And ideally, at the smaller screen sizes, if the user is using the mouse and hovering over the icon, the tooltip behavior for entering and leaving would be the same as at larger screen sizes.

**Review by nick-torres — APPROVED** [review]

**Comment by nick-torres** [comment]
@chasls2 this was merged by accident as you removed the do not merge label, after talking with brian it was my understanding that you were supposed to address his comments during code freeze week so you'll have to make another PR after you address his comments

**Pull request merged**